src/dev/flang/ast/Block.java

```diff
@@ -170,6 +170,18 @@ public UnresolvedType asParsedType()
     return _expressions.size() == 1 ? _expressions.getFirst().asParsedType() : null;
   }
 
+
+  /*
+   * Returns either result of asParsedType()
+   * or Types.t_UNDEFINED in case types name is '_'.
+   */
+  @Override
+  public AbstractType asType()
+  {
+    return _expressions.size() == 1 ? _expressions.getFirst().asType() : null;
+  }
+
+
```

Comment:
This does not seem to be needed since `Expr.asType()` already calles `asParsedType`.